src/ActionModal/components/ActionModal.vue

```diff
@@ -12,173 +12,157 @@
         <span class="action-modal-title">
           {{ requiresSignIn ? `Sign in required` : title }}
         </span>
-        <Steps :steps="['Details', 'Fees', 'Sign']" :active-step="step" />
-      </div>
-      <div v-if="requiresSignIn" class="action-modal-form">
-        <p>You need to sign in to submit a transaction.</p>
-      </div>
-      <div v-else-if="step === `details`" class="action-modal-form">
-        <slot />
-      </div>
-      <div v-else-if="step === `fees`" class="action-modal-form">
-        <TmFormGroup
-          v-if="session.experimentalMode"
-          :error="$v.gasPrice.$error && $v.gasPrice.$invalid"
-          class="action-modal-group"
-          field-id="gasPrice"
-          field-label="Gas Price"
-        >
-          <span class="input-suffix">{{ bondDenom | viewDenom }}</span>
-          <TmField
-            id="gas-price"
-            v-model="gasPrice"
-            step="0.000000001"
-            type="number"
-            min="0"
-          />
-          <TmFormMsg
-            v-if="balanceInAtoms === 0"
-            :msg="`doesn't have any ${bondDenom}s`"
-            name="Wallet"
-            type="custom"
-          />
-          <TmFormMsg
-            v-else-if="$v.gasPrice.$error && !$v.gasPrice.required"
-            name="Gas price"
-            type="required"
-          />
-          <TmFormMsg
-            v-else-if="$v.gasPrice.$error && !$v.gasPrice.between"
-            :max="$v.gasPrice.$params.between.max"
-            :min="0"
-            name="Gas price"
-            type="between"
-          />
-        </TmFormGroup>
-        <TableInvoice
-          :amount="Number(amount)"
-          :estimated-fee="estimatedFee"
-          :bond-denom="bondDenom"
-        />
-        <TmFormMsg
-          v-if="$v.invoiceTotal.$invalid"
-          name="Total"
-          type="between"
-          min="0"
-          :max="balanceInAtoms"
+        <Steps
+          :steps="['Details', 'Fees', 'Sign', 'Send']"
```

Comment:
i think we need a solution that doesn't have 4 steps:

1. user entered *details*
2. show LiTx above *fees*
3. approve (this is slightly different for ledger + extension)
3a. detect and sign on external device
3b. waiting for inclusion
3c. success / failure (retry)

Reply:
Having 3 subpoints for 3. hits my alarm